ItemsListApp/test/ItemsListApp.Api.UnitTests/ItemsControllerUnitTests.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using ItemsListApp.Api.Controllers;
+using ItemsListApp.Api.Models;
+using NUnit.Framework;
+using Assert = NUnit.Framework.Assert;
+
+namespace ItemsListApp.Api.UnitTests
+{
+    [TestFixture]
+    public class ItemsControllerUnitTests
+    {
+        private ItemsController _controller;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _controller = new ItemsController
+            {
+                Request = new HttpRequestMessage(),
+                ControllerContext = {Configuration = new HttpConfiguration()},
+            };
+        }
+
+        [Test]
+        public async Task TestGet()
+        {
+            var expected = new Item
+            {
+                Id = Guid.NewGuid(),
+                Text = "Text of required item",
+            };
+
+            var action = await _controller.GetAsync(expected.Id);
+            var response = await action.ExecuteAsync(CancellationToken.None);
+
```

Comment:
I understand why you placed an empty row here, however, test method should consist of 3 parts (Arrange-Act-Assert) and these should be only free-line-separated blocks in the test method – provided one of them is too long or too repetitive, one may decide to extract a private method from them…